CMakeLists.txt

```diff
@@ -21,8 +41,16 @@ target_link_libraries(boost_sort
     Boost::type_traits
 )
 
+# Testing
+
```

Comment:
I personally dislike such comments. `if(BUILD_TESTING` already provides this information so I see no added value.